addons/escoria-core/game/core-scripts/esc/esc_event_manager.gd

```diff
@@ -31,6 +31,7 @@ const EVENT_ROOM_SELECTOR = "room_selector"
 const EVENT_SETUP = "setup"
 const EVENT_TRANSITION_IN = "transition_in"
 const EVENT_TRANSITION_OUT = "transition_out"
+const EVENT_CANT_REACH = "cant_reach"
```

Comment:
So according to this const, the event to use is `:cant_reach`, not `:cantreach` as you explain in the PR description?

Reply:
@StraToN Sorry, I changed it from "cantreach" to "cant_reach" during testing to make it more readable. I've updated the issue's description to correct this.